src/mpid/ch4/netmod/ofi/ofi_pre.h

```diff
@@ -227,7 +227,7 @@ typedef struct {
                                          * One AVL tree per process. */
     MPL_gavl_tree_t dwin_mrs;   /* Single AVL tree to store locally attached MRs */
 
-    /* Accumulate related info. The struct internally uses MPIDI_OFI_DT_SIZES
+    /* Accumulate related info. The struct internally uses MPIR_DATATYPE_N_PREDEFINED
```

Comment:
this is a good change. glad to get rid of this private, hard to understand constant.